### PR TITLE
New package: ctwm-4.0.2

### PR DESCRIPTION
**Pull request by joshuakraemer**

### CODE REVIEW

**Comment by Gottox** [comment]
Thanks for your contribution! Please use `<pkgname>: <description>` for changes on packages. :+1:

File: srcpkgs/ctwm/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'ctwm'
+pkgname=ctwm
+version=4.0.2
+revision=1
+build_style=cmake
+configure_args="-DUSE_JPEG=OFF"
+hostmakedepends="flex m4"
+makedepends="libX11-devel libXext-devel libXmu-devel libXt-devel libSM-devel libICE-devel libXpm-devel libXrandr-devel"
+depends="m4"
```
Comment (jnbr) [inline srcpkgs/ctwm/template]: is `m4` realy a runtime dependency of `ctwm`

**Comment by jnbr** [comment]
Please squash your commits to a single one with commit message: "New package: ctwm-4.0.2."

File: srcpkgs/ctwm/template (hunk 1 above)
Comment (joshuakraemer) [reply srcpkgs/ctwm/template]: Yes, it is used to preprocess config files. I have added m4 and other optional dependencies as build options.

Comment (jnbr) [reply srcpkgs/ctwm/template]: Ok. Thank you for clarifying. Looked suspicious.

**Pull request merged**